S4TF_Tutorial_2.ipynb

```diff
@@ -0,0 +1,878 @@
+        "\n",
+        "Could you determine the relationship between the two *without* using machine learning?  That is, could you use traditional programming techniques (for example, a lot of conditional statements) to create a model?  Perhaps—if you analyzed the dataset long enough to determine the relationships between temperatures. And this becomes difficult—maybe impossible—on more complicated datasets. A good machine learning approach *determines the model for you*. If you feed enough representative examples into the right machine learning model type, the program will figure out the relationships for you.\n",
+        "\n",
+        "### Select the model\n",
+        "\n",
+        "We need to select the kind of model to train. We will use simplest possible model we can, a Dense network. Since the problem is straightforward, this network will require only a single layer, with a single neuron. \n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "EycLbiRvt7wE",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Create a model using the Swift for TensorFlow Deep Learning Library\n",
+        "\n",
+        "The [Swift for TensorFlow Deep Learning Library](https://github.com/tensorflow/swift-apis) defines primitive layers and conventions for wiring them together, which makes it easy to build models and experiment.\n",
+        "\n",
+        "A model is a `struct` that conforms to [`Layer`](https://www.tensorflow.org/swift/api_docs/Protocols/Layer), which means that it defines a [`call(_:)`](https://www.tensorflow.org/swift/api_docs/Protocols/Layer#call_:) method that maps input `Tensor`s to output `Tensor`s. The `call(_:)` method often simply sequences the input through sublayers. Let's define an `c2fConv` that sequences the input through a single [`Dense`](https://www.tensorflow.org/swift/api_docs/Structs/Dense) layer.\n",
+        "\n",
+        "*   `inputSize:1` — This specifies that the input to this layer is a single value. Since this is the first (and only) layer, that input shape is the input shape of the entire model. The single value is a Tensor matrix of floating point numbers, representing degrees Celsius.\n",
+        "\n",
+        "* `outputSize:1` — This specifies that the output of this layer is a single value. The single value is a tensor matrix of floating point numbers, representing degrees Fahrenheit."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "VNF9rry4biBh",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "import TensorFlow\n",
```

Comment:
This second `import TensorFlow` can probably be removed, because it was already imported above.

Reply:
Oh yes sure 👍